app/models/compliance.rb

```diff
@@ -83,7 +83,7 @@ def self.check_compliance(target, _inputs = {})
       return
     end
 
-    compliance_result = results[:actions].nil? ? true : !results[:actions].key?(:compliance_failed)
+    compliance_result = results[:action].nil? ? true : !results[:action].key?(:compliance_failed)
```

Comment:
@jrafanie you're right. This is a typo introduced by [`e5f2470`](https://github.com/ManageIQ/manageiq/commit/e5f24705b8f8b2b5ec1cd8d20b93e8d9503d3420) as you found out. The fix should be there in [miq_policy.rb](https://github.com/ManageIQ/manageiq/commit/e5f24705b8f8b2b5ec1cd8d20b93e8d9503d3420#diff-74bb73b4bf1317b7b3582b5b962c88bcR190), something like this
```
    actions = invoke_actions(target, mode, profiles, succeeded, failed, inputs.merge(:event => erec))
    result[:actions] = actions if actions
    result
```